packages/devtools_app/lib/src/inspector/flutter/inspector_service_flutter_extension.dart

```diff
@@ -96,18 +99,18 @@ extension InspectorFlutterService on ObjectGroup {
                   };
                   if (constraints is BoxConstraints) {
                     constraintsProperty.addAll(<String, Object>{
-                      'minWidth': constraints.minWidth.toStringAsFixed(1),
-                      'minHeight': constraints.minHeight.toStringAsFixed(1),
-                      'maxWidth': constraints.maxWidth.toStringAsFixed(1),
-                      'maxHeight': constraints.maxHeight.toStringAsFixed(1),
+                      'minWidth': toStringAsFixed(constraints.minWidth),
```

Comment:
just use toString and add epsilon logic in the devtools code to handle rounding.